tests/foreman/cli/test_computeresource_vmware.py

```diff
@@ -77,10 +78,10 @@ def test_positive_vmware_cr_end_to_end(target_sat, module_org, module_location,
 @pytest.mark.e2e
 @pytest.mark.on_premises_provisioning
 @pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete=True'], indirect=True)
-@pytest.mark.parametrize('vmware', ['vmware7', 'vmware8'], indirect=True)
-@pytest.mark.parametrize('pxe_loader', ['bios', 'uefi'], indirect=True)
-@pytest.mark.parametrize('provision_method', ['build', 'bootdisk'])
-@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.parametrize('vmware', ['vmware8'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['uefi'], indirect=True)
+@pytest.mark.parametrize('provision_method', ['build'])
+@pytest.mark.rhel_ver_match('[8, 9]')
```

Comment:
i noticed you have a hardcoded `rhel8` here:
https://github.com/SatelliteQE/robottelo/pull/14094/files#diff-f5876fe03942ff6e50c18a8f2c95255cdca6f84b9956721a587abf7322fb73abR130
mind changing that, so it reflects the current rhel_ver parameter?